src/js/DrawStyleController.js

```diff
@@ -189,8 +212,10 @@ goog.require('ga_styles_service');
     $scope.options.textColor = options.colors[5];
     $scope.options.textSize = options.textSizes[0];
     $scope.options.icon = options.icons[0];
+    $scope.options.iconsBabs = getImgsByCategoryName('babs');
```

Comment:
Try to use something more generic,  keep in mind that the next time we want to add a category, we only want to add a element in the iconCategories object. 
Ex: 
```
foreach (category) {
    if (!category.useColorOption) {
        category.icons =  getIconsByCategory(category);
    } 
}
```